python/lsst/ctrl/mpexec/cli/script/run.py

```diff
@@ -159,6 +161,12 @@ def run(  # type: ignore
         If `True`, do not disable implicit threading by third-party libraries.
         Implicit threading is always disabled during actual quantum execution
         if ``processes > 1``.
+    cores_per_quantum : `int`
+        Number of cores that can be used by each quantum. Defaults to 1.
```

Comment:
There is no default for that argument.

Reply:
Ah. Thanks. The defaulting is above and below but in this API everything is mandatory...